src/Ryujinx.Ava/UI/Views/Input/MotionInputView.axaml.cs

```diff
@@ -6,23 +6,22 @@
 using Ryujinx.Common.Configuration.Hid.Controller;
 using System.Threading.Tasks;
 
-namespace Ryujinx.Ava.UI.Windows
+namespace Ryujinx.Ava.UI.Views.Input
 {
-    public partial class MotionSettingsWindow : UserControl
+    public partial class MotionInputView : UserControl
     {
-        private readonly InputConfiguration<GamepadInputId, StickInputId> _viewmodel;
+        private MotionInputViewModel ViewModel;
```

Comment:
```suggestion
        private MotionInputViewModel _viewModel;
```
Private field should start with `_`